hedera-node/src/main/java/com/hedera/services/store/models/NftId.java

```diff
@@ -36,6 +36,13 @@ public NftId(long shard, long realm, long num, long serialNo) {
 		this.serialNo = serialNo;
 	}
 
+	public NftId(long num, long serialNo) {
```

Comment:
I think it's better to remove this constructor and not hardcode shard and realm to 0s. In the future those fields can have a different value and then it wouldn't be correct to assume they're zeros.
Instead, I would suggest to add a static method in EntityIdUtils: `asNftId(Id tokenId, long serialNumer)` which calls the existing NftId constructor with the realm and shard values of the Id.